test_fakeredis.py

```diff
@@ -159,15 +158,15 @@ def test_future_newbytes(self):
         try:
             from builtins import bytes
         except ImportError:
-            raise SkipTest('future.types not available')
+            pytest.skip('future.types not available')
```

Comment:
I think this could be simplified using [pytest.importorskip](https://doc.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency).